.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
If a "git noob" sends a PR with the whole `.idea` folder checked in, it would be a great opportunity to teach them a bit more about Git 👍 

But no, it's probably not worth discussing any further. I told you my preference, you decide.